### PR TITLE
Add stm32l462ve

### PR DESCRIPTION
**Pull request by heezes**
Added two more MCU's to the list
[#]Stm32l462ve
[#]Bluenrg2

### CODE REVIEW

File: project_generator_definitions/mcu/st/bluenrg2.yaml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+mcu:
+    vendor:
```
Comment (0xc0170) [inline project_generator_definitions/mcu/st/bluenrg2.yaml]: this is not MCU , not required to be here. you can create our own project with this board and just use the MCU you are adding below ?

File: project_generator_definitions/mcu/st/stm32l462ve.yaml [hunk 2]
```diff
@@ -0,0 +1,7 @@
+mcu:
+    vendor:
+        - st
+    name:
+        - stm32l462ve
+    core:
+        - cortex-m4
```
Comment (0xc0170) [inline project_generator_definitions/mcu/st/stm32l462ve.yaml]: can you add a new line here

**Review by 0xc0170 — CHANGES_REQUESTED** [review]

**Comment by heezes** [comment]
@0xc0170 Recent commit has the requested changes

**Review by 0xc0170 — APPROVED** [review]

**Comment by 0xc0170** [comment]
Travis is not running, they changed their plans. I'll create an issue and move it elsewhere 🤦

**Pull request merged**